src/ocrd_utils/config.py

```diff
@@ -141,6 +143,18 @@ def _ocrd_download_timeout_parser(val):
     description="Timeout in seconds for connecting or reading (comma-separated) when downloading.",
     parser=_ocrd_download_timeout_parser)
 
+config.add("OCRD_DOWNLOAD_INPUT",
+    description="Whether to download files not present locally during processing",
+    default=(True, True),
+    validator=_validator_boolean,
+    parser=_parser_boolean)
+
+config.add("OCRD_MISSING_INPUT",
+    description="How to deal with missing input files (for some fileGrp/pageId) during processing [SKIP|ABORT]",
+    default=(True, 'SKIP'),
+    validator=lambda val: val in ['SKIP', 'ABORT'],
```

Comment:
What about FIRST and LAST? Or are these supposed to be only internals?

Reply:
Does not apply here. Remember `first` and `last` are only for the internal problem of dealing with non-unique matches. We did not want to bother the user with this level.